packages/plugin-ext/src/main/browser/style/tree.css

```diff
@@ -44,3 +44,28 @@
     margin-bottom: 10px;
     margin-left: 17px;
 }
+
+.theia-tree-view .theia-TreeContainer .theia-WelcomeViewInfo {
+    padding: 7px 20px 0 20px;
```

Comment:
We should not use hardcoded constants but rather use variables.
The same applies to other css updates included in the pull-request.